### ISSUE TITLE
Accessibility Issues with Pluto.jl

### ISSUE THREAD

**Issue by vquintanilla0911** [fonsp/Pluto.jl#2485]
Hello,

We are conducting an accessibility evaluation of Pluto.jl for use with a college district and it appears it is not accessible for users with disabilities.

The Wave Accessibility tool indicated color-contrast issues, missing alternative texts on images, and missing page titles.

Is this something that could be remediated/fixed?

<img width="1719" alt="Pluto Accessibility Test 2" src="https://user-images.githubusercontent.com/109389074/221992848-60cb370b-4d7f-410e-9107-662420f2a00a.png">
<img width="1719" alt="Pluto Accessibility Test" src="https://user-images.githubusercontent.com/109389074/221992852-69a062d1-d8e5-41d6-a0f6-ad1074852359.png">

**Comment by pankgeorg** [comment]
Yes, we can prioritize and address these issues. We are also interested in usability concerns that may not appear in this or other tools. If you or someone you know uses pluto on a accessibility enabled device, we'd love to get firsthand feedback!

We meet and discuss every Tuesday at 18.00 CET.

Thanks!
